### ISSUE TITLE
docker-compose.yml Reserved character '@' is not allowed outside quotes

### ISSUE THREAD

**Issue by overpod** [graphcool/prisma#2263]
When you start a local cluster.
I have a mysql user of the form "test @ test"
Cluster 1.7 docker-compose
### Error
``` bash
Exception in thread "main" com.typesafe.config.ConfigException$Parse: String: 7: Expecting close brace } or a comma, got '@' (Reserved character '@' is not allowed outside quotes) (if you intended '@' (Reserved character '@' is not allowed outside quotes) to be part of a key or string value, try enclosing the key or value in double quotes, or you may be able to rename the file .properties rather than .conf)
```

**Comment by marktani** [comment]
Can you show your entire `docker-compose.yml` file?

**Comment by overpod** [comment]
```yml
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.7
    restart: always
    ports:
    - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        port: 4466
        databases:
          default:
            connector: mysql
            active: true
            host: abuga.mysql.database.azure.com
            port: 3306
            user: ""trofimov@abuga""
            password: ********
            managementApiSecret: ******
```

**Comment by overpod** [comment]
```bash
docker-compose up
```

**Comment by overpod** [comment]
```yml
            active: true
            host: abuga.mysql.database.azure.com
            port: 3306
            "user: trofimov@abuga"
```
### Error
```bash
prisma_1  | Exception in thread "main" java.lang.RuntimeException: Unable to load Prisma config: while scanning a simple key
prisma_1  |  in 'string', line 9, column 5:
prisma_1  |         'user: trofimov@abuga'
prisma_1  |         ^
prisma_1  | could not find expected ':'
prisma_1  |  in 'string', line 10, column 5:
prisma_1  |         password: 'ZXcasdqwe123'
prisma_1  |         ^
prisma_1  |
prisma_1  |     at scala.sys.package$.error(package.scala:27)
prisma_1  |     at com.prisma.config.ConfigLoader$.load(ConfigLoader.scala:38)
prisma_1  |     at com.prisma.local.PrismaLocalDependencies.<init>(PrismaLocalDependencies.scala:38)
prisma_1  |     at com.prisma.local.PrismaLocalMain$.delayedEndpoint$com$prisma$local$PrismaLocalMain$1(PrismaLocalMain.scala:15)
prisma_1  |     at com.prisma.local.PrismaLocalMain$delayedInit$body.apply(PrismaLocalMain.scala:12)
prisma_1  |     at scala.Function0.apply$mcV$sp(Function0.scala:34)
prisma_1  |     at scala.Function0.apply$mcV$sp$(Function0.scala:34)
prisma_1  |     at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
prisma_1  |     at scala.App.$anonfun$main$1$adapted(App.scala:76)
prisma_1  |     at scala.collection.immutable.List.foreach(List.scala:389)
prisma_1  |     at scala.App.main(App.scala:76)
prisma_1  |     at scala.App.main$(App.scala:74)
prisma_1  |     at com.prisma.local.PrismaLocalMain$.main(PrismaLocalMain.scala:12)
prisma_1  |     at com.prisma.local.PrismaLocalMain.main(PrismaLocalMain.scala)
prisma_prisma_1 exited with code 1
```

**Comment by marktani** [comment]
I'm sorry, I meant `user: "trofimov@abuga"`.

**Comment by overpod** [comment]
```bash
trofimov@abuga:~/projects/prisma$ cat docker-compose.yml
version: '3'
services:
  prisma:
    image: prismagraphql/prisma:1.7
    restart: always
    ports:
    - "4466:4466"
    environment:
      PRISMA_CONFIG: |
        clusterAddress: "https://prisma.abuga.ru"
        port: 4466
        databases:
          default:
            connector: mysql
            active: true
            host: abuga.mysql.database.azure.com
            port: 3306
            user: "trofimov@abuga"
            password: "*******"
```
```bash
trofimov@abuga:~/projects/prisma$ docker-compose up
Recreating prisma_prisma_1 ... done
Attaching to prisma_prisma_1
prisma_1  | Exception in thread "main" com.typesafe.config.ConfigException$Parse: String: 7: Expecting close brace } or a comma, got '@' (Reserved character '@' is not allowed outside quotes) (if you intended '@' (Reserved character '@' is not allowed outside quotes) to be part of a key or string value, try enclosing the key or value in double quotes, or you may be able to rename the file .properties rather than .conf)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseError(ConfigDocumentParser.java:201)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseError(ConfigDocumentParser.java:197)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseObject(ConfigDocumentParser.java:526)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseValue(ConfigDocumentParser.java:247)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseObject(ConfigDocumentParser.java:456)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseValue(ConfigDocumentParser.java:247)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parseObject(ConfigDocumentParser.java:456)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser$ParseContext.parse(ConfigDocumentParser.java:646)
prisma_1  |     at com.typesafe.config.impl.ConfigDocumentParser.parse(ConfigDocumentParser.java:14)
prisma_1  |     at com.typesafe.config.impl.Parseable.rawParseValue(Parseable.java:260)
prisma_1  |     at com.typesafe.config.impl.Parseable.rawParseValue(Parseable.java:248)
prisma_1  |     at com.typesafe.config.impl.Parseable.parseValue(Parseable.java:180)
prisma_1  |     at com.typesafe.config.impl.Parseable.parseValue(Parseable.java:174)
prisma_1  |     at com.typesafe.config.impl.Parseable.parse(Parseable.java:299)
prisma_1  |     at com.typesafe.config.ConfigFactory.parseString(ConfigFactory.java:1001)
prisma_1  |     at com.typesafe.config.ConfigFactory.parseString(ConfigFactory.java:1011)
prisma_1  |     at com.prisma.deploy.connector.mysql.InternalDatabaseDefs.typeSafeConfigFromDatabaseConfig(InternalDatabaseDefs.scala:41)
prisma_1  |     at com.prisma.deploy.connector.mysql.InternalDatabaseDefs.database(InternalDatabaseDefs.scala:20)
prisma_1  |     at com.prisma.deploy.connector.mysql.InternalDatabaseDefs.internalDatabase$lzycompute(InternalDatabaseDefs.scala:12)
prisma_1  |     at com.prisma.deploy.connector.mysql.InternalDatabaseDefs.internalDatabase(InternalDatabaseDefs.scala:12)
prisma_1  |     at com.prisma.deploy.connector.mysql.MySqlDeployConnector.internalDatabase$lzycompute(MySqlDeployConnector.scala:19)
prisma_1  |     at com.prisma.deploy.connector.mysql.MySqlDeployConnector.internalDatabase(MySqlDeployConnector.scala:19)
prisma_1  |     at com.prisma.deploy.connector.mysql.MySqlDeployConnector.<init>(MySqlDeployConnector.scala:22)
prisma_1  |     at com.prisma.connectors.utils.ConnectorUtils$.loadDeployConnector(ConnectorUtils.scala:25)
prisma_1  |     at com.prisma.local.PrismaLocalDependencies.deployConnector$lzycompute(PrismaLocalDependencies.scala:95)
prisma_1  |     at com.prisma.local.PrismaLocalDependencies.deployConnector(PrismaLocalDependencies.scala:95)
prisma_1  |     at com.prisma.deploy.DeployDependencies.initialize(DeployDependencies.scala:38)
prisma_1  |     at com.prisma.deploy.DeployDependencies.initialize$(DeployDependencies.scala:37)
prisma_1  |     at com.prisma.local.PrismaLocalDependencies.initialize(PrismaLocalDependencies.scala:31)
prisma_1  |     at com.prisma.local.PrismaLocalMain$.delayedEndpoint$com$prisma$local$PrismaLocalMain$1(PrismaLocalMain.scala:17)
prisma_1  |     at com.prisma.local.PrismaLocalMain$delayedInit$body.apply(PrismaLocalMain.scala:12)
prisma_1  |     at scala.Function0.apply$mcV$sp(Function0.scala:34)
prisma_1  |     at scala.Function0.apply$mcV$sp$(Function0.scala:34)
prisma_1  |     at scala.runtime.AbstractFunction0.apply$mcV$sp(AbstractFunction0.scala:12)
prisma_1  |     at scala.App.$anonfun$main$1$adapted(App.scala:76)
prisma_1  |     at scala.collection.immutable.List.foreach(List.scala:389)
prisma_1  |     at scala.App.main(App.scala:76)
prisma_1  |     at scala.App.main$(App.scala:74)
prisma_1  |     at com.prisma.local.PrismaLocalMain$.main(PrismaLocalMain.scala:12)
prisma_1  |     at com.prisma.local.PrismaLocalMain.main(PrismaLocalMain.scala)
prisma_prisma_1 exited with code 1
```

**Comment by overpod** [comment]
How to escape a character @ ?
I consider this behavior a mistake !

**Comment by overpod** [comment]
All problems are solved version 1.7.3. Thank you

**Comment by overpod** [comment]
This problem needs to be closed.
